### PR TITLE
Documentation/Bugfix: release v10, make tests green and add payment transaction data

### PR DESCRIPTION
**Pull request by jdsika**
General:

- [x] Transaction Documentation
- [x] Release Notes
- [x] Make green tests and fix bugs
- [x] Add failsafe to avoid usage after 01.04.2022

Contributor | Sum hours | $ / hour | $ | VWAP XTZUSD | XTZ | missing | XTZ | hash
-- | -- | -- | -- | -- | -- | -- | -- | --
@amzid | 2 | 70 | 140 | 3.76145377668336 | 37.2196518452087 |   | 37.2196518452087 | [link](https://tzkt.io/opPS5QpPUyDxFzZ6MCdmtxwah7ZMSGRxQzFRgPqXC5thP5SF4j7)
@dansan566  | 1 | 70 | 70 | 3.76145377668336 | 18.6098259226044 | 139.554445 | 158.164270922604 | [link](https://tzkt.io/opU94ETdKbpRrLwW57Fqb79FbAJynJRYyCSW6o4udTw7pToNcaX)
@denver-s | 10 | 70 | 700 | 3.76145377668336 | 186.098259226044 |   | 186.098259226044 | Waiting for grant
@ericlavoie | 15 | 70 | 1050 | 3.76145377668336 | 279.147388839065 |   | 279.147388839065 | Waiting for grant
@jdsika | 68.5 | 70 | 4795 | 3.76145377668336 | 1274.7730756984 |   | 1274.7730756984 | [link](https://tzkt.io/onwJpuEWxRTSV2xrWZCgrSz78tgpq8FgxZ6wzAyvMSFwPCp9giM)
@Pea-skillz | 5 | 70 | 350 | 3.76145377668336 | 93.0491296130218 |   | 93.0491296130218 | [link](https://tzkt.io/ooDEaVtPoVFgrQybEMWcPfni3eqQZKjqyvusmk1uXtiB2xVvkSn)
@utdrmac | 12.5 | 70 | 875 | 3.76145377668336 | 232.622824032554 |   | 232.622824032554 | [link](https://tzkt.io/onrZkPDTCKeEHzFrzFRbh7XNrwGvBQAg8pw3g9PSusTJTaQ6CJQ)
@vkresch | 31 | 70 | 2170 | 3.76145377668336 | 576.904603600735 |   | 576.904603600735 | [link](https://tzkt.io/oo8YqiZCQUNqxDv2bH9RV69CskznDgZWVm61EZ9q2PdNC8gXeED )

Effort=4h

### CODE REVIEW

**Comment by jdsika** [comment]
@nicolasochem do you have an idea for an easy way for a graceful shutdown after Ithaca activation on the master branch?

File: src/main.py [hunk 1]
```diff
@@ -2,8 +2,11 @@
 import sys
 import pip
 import pkg_resources
+from datetime import date
+
 
 REQUIREMENTS_FILE_PATH = "requirements.txt"
+END_OF_SERVICE = date(2022, 4, 1)
```
Comment (nicolasochem) [inline src/main.py:9]: maybe april 3rd? it is still ok in the very first ithaca cycle since people will be paying the last hangzhou cycle

**Review by nicolasochem — COMMENTED** [review]
few questions

File: src/util/csv_payment_file_parser.py [hunk 2]
```diff
@@ -9,7 +9,7 @@ def __init__(self) -> None:
         super().__init__()
 
     def parse(self, payment_report_file, cycle):
-        with open(payment_report_file) as f:
+        with open(payment_report_file, newline="") as f:
             # read csv into list of dictionaries
```
[the remark this answers is not in the document]
Comment (jdsika) [reply src/util/csv_payment_file_parser.py:13]: Without it created empty lines on windows. I will test again on ubuntu right away but I think it works...

File: src/main.py (hunk 1 above)
Comment (jdsika) [reply src/main.py:9]: people can modify the date themselves?

**Review by nicolasochem — APPROVED** [review]

**Pull request merged**